lib/cryptocoin_payable/commands/payment_processor.rb

```diff
@@ -4,6 +4,34 @@ def self.perform
       new.perform
     end
 
+    def self.update_transactions_for(payment)
```

Comment:
I've moved this to its own public method because it may be useful to be able to explicitly sync transactions for a payment. In my own app I use BlockCypher's confidence interval web hook which allows me to immediately set a payment to confirmed state. But upon doing so, transactions should also be synced.

(Once this PR is merged I will add full support for BlockCypher web hooks to the gem)